### PR TITLE
Improvement: Restrict filetypes in custom fonts filearea, solves #120.

### PR DESCRIPTION
**Pull request by abias**

### CODE REVIEW

File: locallib.php [hunk 1]
```diff
@@ -692,3 +692,60 @@ function theme_boost_union_get_customfonts_templatecontext() {
 
     return $filesforcontext;
 }
+
+/*
```
Comment (lucaboesch) [inline locallib.php:696]: Put two stars here
/ **

File: locallib.php [hunk 2]
```diff
@@ -692,3 +692,60 @@ function theme_boost_union_get_customfonts_templatecontext() {
...
+ * Helper function which makes sure that all webfont file types are registered in the system.
+ * The webfont filetypes need to be registered in the system, otherwise the admin settings filepicker wouldn't allow restricting
+ * the uploadable filetypes to webfonts only. *
```
Comment (lucaboesch) [inline locallib.php:699]: Remove the star at the end of the line.

Comment (lucaboesch) [inline locallib.php:699]:
```
 * @return void
 * @throws coding_exception

```
do help here (not sure about the @returns but the @throws has to be)

File: locallib.php [hunk 3]
```diff
@@ -692,3 +692,60 @@ function theme_boost_union_get_customfonts_templatecontext() {
...
+ */
+function theme_boost_union_register_webfonts_filetypes() {
+    // Our array of webfonts to register.
```
Comment (lucaboesch) [inline locallib.php:702]: Better would be:
`    // Our array of webfonts filetypes to register.`

**Comment by abias** [comment]
Thank you, Luca, for your quick review.

I managed to fix all raised items and pushed another version.

**Review by lucaboesch — COMMENTED** [review]
That's good now.
Merging.

**Review by lucaboesch — APPROVED** [review]

**Pull request merged**